src/lib/icons/play_48px.svg

```diff
@@ -1,4 +1,5 @@
-<svg fill="#fff" height="48" viewBox="0 0 24 24" width="48" focusable="false">
+<svg fill="#fff" height="48" viewBox="0 0 24 24" width="48" focusable="false" role="img" aria-labelledby="play-button-icon-01">
+    <title id="play-button-icon-01"></title>
```

Comment:
We should avoid using `id` for selectors since multiple instances of Preview can be rendered in a document at once.